app/controllers/companies_controller.rb

```diff
@@ -1,6 +1,9 @@
 # frozen_string_literal: true
 
 require 'gpw/client'
+require 'gpw/application_service'
+require 'gpw/companies_list_updator'
+require 'gpw/companies_details_updator'
```

Comment:
do you need it? I thought that it will be loaded automatically since you keep these files in `app/lib`

Reply:
Of course now :)